packages/empty-state/src/BasicEmptyState/BasicEmptyState.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Button from '@leafygreen-ui/button';
+import LeafyGreenProvider from '@leafygreen-ui/leafygreen-provider';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import { isComponentType } from '@leafygreen-ui/lib';
+import { Body, H3, Link } from '@leafygreen-ui/typography';
+
+import {
+  buttonContainerStyles,
+  descriptionStyles,
+  externalLinkStyles,
+  rootStyles,
+  textContainerStyles,
+  titleStyles,
+} from './BasicEmptyState.styles';
+import { BasicEmptyStateProps } from '.';
+
+export function BasicEmptyState({
```

Comment:
Should we make this a forwardRef?

Reply:
true, this component might not necessarily need a frowardedRef, but going forward with new components, we should make them all have forwardedRefs to be consistent.